qiskit/transpiler/passes/layout/vf2_post_layout.py

```diff
@@ -51,25 +51,26 @@ class VF2PostLayout(AnalysisPass):
     """A pass for choosing a Layout after transpilation of a circuit onto a
     Coupling graph, as a subgraph isomorphism problem, solved by VF2++.
 
-    Unlike the :class:`~.VF2PostLayout` transpiler pass which is designed to find an
+    Unlike the :class:`~.VF2Layout` transpiler pass which is designed to find an
     initial layout for a circuit early in the transpilation pipeline this transpiler
     pass is designed to try and find a better layout after transpilation is complete.
     The initial layout phase of the transpiler doesn't have as much information available
     as we do after transpilation. This pass is designed to be paired in a similar pipeline
     as the layout passes. This pass will strip any idle wires from the circuit, use VF2
     to find a subgraph in the coupling graph for the circuit to run on with better fidelity
-    and then update the circuit layout to use the new qubits.
+    and then update the circuit layout to use the new qubits. The algorithm used in this
+    pass is described in `arXiv:2209.15512 <https://arxiv.org/abs/2209.15512>`__.
 
-    If a solution is found that means there is a "perfect layout" and that no
-    further swap mapping or routing is needed. If a solution is found the layout
-    will be set in the property set as ``property_set['layout']``. However, if no
-    solution is found, no ``property_set['layout']`` is set. The stopping reason is
+    If a solution is found that means there is a lower error layout available for the
```

Comment:
```suggestion
    If a solution is found that means there is a low error layout available for the
```

Reply:
I think this is actually better as is. The `VF2PostLayout` pass is searching for a lower estimate error rate subgraph than the current layout. It's not guaranteed to find a low error rate in general it actually could still be quite high it's just relative to the input layout when the pass is run.